src/main/java/org/home/implementation/FileMessageStorageImpl.java

```diff
@@ -1,25 +1,101 @@
 package org.home.implementation;
 
 import org.home.interfaces.MessageStorageInterface;
+import org.home.model.FileMessage;
 import org.home.model.Message;
 import org.home.model.MessageQueue;
 import org.springframework.stereotype.Component;
 
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.util.ArrayList;
 import java.util.Queue;
 
-@Component("secondary")
+//@Component("secondary")
 public class FileMessageStorageImpl implements MessageStorageInterface {
-    public FileMessageStorageImpl() {
+    String QFileName;
```

Comment:
Why QFileName should be stored inside of FileHandler ?
A FileHandler should not understand what is Queue and all!! The logic of Handling File should be here. 

Think about it